Listener/ExceptionListener.php

```diff
@@ -23,7 +24,7 @@
 class ExceptionListener
 {
     /**
-     * @var array
+     * @var \Exception[]
```

Comment:
That's not true, this is a mapping of `className` to a pair `(code, expose_message)`.